data/chapters.json

```diff
@@ -27,6 +27,13 @@
     },
     "title": "St. Louis"
   },
+,
+  "kochi": {
+    "dataService": {
+      "id": "papers-we-love-kochi",
+    },
```

Comment:
Also, are you using Meetup.com?

Reply:
Presently, no. Given the circle within Kochi. Most event publicity happens via telegram. However, we just have had three meetups so far. We'll set it up soon if we are to.